spec/support/helpers.rb

```diff
@@ -99,9 +99,6 @@ def bundle(cmd, options = {})
       with_sudo = options.delete(:sudo)
       sudo = with_sudo == :preserve_env ? "sudo -E" : "sudo" if with_sudo
 
-      no_color = options.delete("no-color") { cmd.to_s !~ /\A(e|ex|exe|exec|conf|confi|config)(\s|\z)/ }
-      options["no-color"] = true if no_color
```

Comment:
Why did you remove this?

Reply:
I was trying to test the command line arguments that `bundle show` receives and for some reason I was getting the `--no-color` flag added to them? Then I found this flag and I added `s|sh|sho|show` as exceptions here and it worked.

But then I decided that doing this didn't make any sense to me, so I removed it so I could get some feedback from you :smiley:. Why are we doing that?

Reply:
So all the test output will never contain ANSI escape codes , and thus be consistent no matter how the tests are being run

Reply:
Can you show an example of this inconsistency? Our current test environment seems to handle this just fine? (I think the explanation of why this just works is that an `open3` context is recognized by ruby as a "non-tty" situation, so it's automatically [turned off](https://github.com/bundler/bundler/blob/e115a839123a21365904fd86980452d2b2b0283e/lib/bundler/ui/shell.rb#L13) by `bundler`).

To me doing this is not good because the helper does not represent real world usage, but a slight modification of it, and it leads to weird assertions (like the ones I'm fixing here), and suprising behavior for the developer (when I first saw this I was like "what? where did this flag come from?", and I had to spend a while tracking it down and adding yet more exceptions here).

Reply:
You could also pass no-color => false to the bundle helper

Reply:
Right. So is that what you're asking me to do?

Reply:
I'm going to do that and make my case to remove this separately :slightly_smiling_face:.

Reply:
This flag will be needed on most `bundle show` specs. Do you prefer that I explicitly pass the flag to every `bundle` invocation that needs it, that I add more exceptions inside the method, or that I leave this patch as it is?

Reply:
It's your call :) If it doesn't break the tests, I suppose removing it is fine?

Reply:
Ok, yeah, I think it's fine!